### PR TITLE
Add urgency to task

### PR DESCRIPTION
**Pull request by kdheepak**
- [x] Add urgency to task
- [x] Write tests

### CODE REVIEW

**Comment by kdheepak** [comment]
@matthiasbeyer This is ready for your review!

File: src/task.rs [hunk 1]
```diff
@@ -493,6 +499,21 @@ impl Task {
...
+        self.urgency.as_ref()
+    }
+
+    /// Get the urgency of the task
+    pub fn urgency_mut(&mut self) -> Option<&mut Urgency> {
+        self.urgency.as_mut()
+    }
+
+    /// Set the urgency of the task
+    pub fn set_urgency<T>(&mut self, new: Option<T>) where T: Into<Urgency> {
```
Comment (matthiasbeyer) [inline src/task.rs:513]: Please add newlines before the `where` clause to keep the style consistent.

File: src/urgency.rs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+
+//! Module containing `Urgency` type
+
+/// type definition for Urgency
+pub type Urgency = f64;
```
Comment (matthiasbeyer) [inline src/urgency.rs:10]: Please add a trailing newline.

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
Looks good apart from the stylistic stuff annotated. Please fix them, then this is ready.

File: src/task.rs (hunk 1 above)
Comment (kdheepak) [reply src/task.rs]: Resolved in 06777f6.

File: src/urgency.rs (hunk 2 above)
Comment (kdheepak) [reply src/urgency.rs:10]: All the other files don't have a trailing newline at the end of the file. Is that what you meant? Do you still want me to add it even though the other files don't have it?

Comment (matthiasbeyer) [reply src/urgency.rs:10]: If they don't have it I'm fine this way.

**Review by matthiasbeyer — APPROVED** [review]

**Comment by kdheepak** [comment]
Would you like me to merge this then?

**Comment by matthiasbeyer** [comment]
I invited you to this repository because I think you can do these things and bring this crate forwards, so yes, feel free.
All I ask you to keep in mind is that the code for this crate does nothing unnecessarily complex or out-of-scope!

Feel free to continue working on this as you see fit. :+1:

**Comment by kdheepak** [comment]
I'll definitely only make changes through PRs, so you can review them whenever you have the time!

**Comment by matthiasbeyer** [comment]
Nice. I added a branch protection for master, so everything has to go through PR and CI must be green. That should be enough!

Thank you very much for contributing to this crate!

**Comment by kdheepak** [comment]
Awesome!

**Pull request merged**